README.md

```diff
@@ -723,25 +724,25 @@ calibration (linear filter) and the infinite impulse response filter in ts.conf.
 
 ### CMake
 
-Alternatively you can use CMake to build the project.
-To create an build in the project tree:
+Alternatively, you can use CMake to build the project.
+To create building in the project tree:
```

Comment:
This ought to be "a build"

Reply:
If you want to fix that and the above replaced sentence on top now, go ahead. Otherwise I might do it soon. thanks!